app/views/collections/_form_for_select_collection.html.erb

```diff
@@ -13,9 +13,10 @@
               <fieldset>
                 <legend><%= t("sufia.collection.select_form.select_heading") %></legend>
               <ul>
-                <% user_collections.sort { |c1,c2| c1['date_modified_dtsi'] <=> c2['date_modified_dtsi'] }.each do |collection|  %>
+                <% user_collections.sort { |c1,c2| c2['system_create_dtsi'] <=> c1['system_create_dtsi'] }.each_with_index do |collection,index|  %>
```

Comment:
using solr_name or `CatalogController.uploaded_field` here would avoid having to know _dtsi.